content/events/_index.md

```diff
@@ -4,4 +4,7 @@ description: Talks, workshops, and other gatherings to help you connect with the
 meta_desc: |
     Pulumi events, both online and in-person, bringing the cloud computing community together
     to connect, collaborate, and learn new techniques and best practices.
+
+aliases:
```

Comment:
I changed the URL for old Kubecon workshop and added an alias here so incase the URL ranking in search people would just be directed to the /events page, instead of the old landing page.